cluster/src/main/java/com/hartwig/pipeline/tools/VersionUtils.java

```diff
@@ -50,6 +51,10 @@ public static String majorMinorVersion(final String version) {
         return version;
     }
 
+    public static List<HmfTool> inArtifactRegistry() {
+        return List.of(HmfTool.ORANGE);
```

Comment:
Assuming this means that ony ORANGE is now retrieved from repo instead of bucket it contradicts your statement in readme that all HMF tools should now be pushed to repo, but maybe adding the rest comes later, in that case fine of course.